cmd/controller-manager/app/controllermanager.go

```diff
@@ -100,17 +100,19 @@ func setupControllers(mgr controllerruntime.Manager, opts *options.Options, stop
 	resetConfig := mgr.GetConfig()
 	dynamicClientSet := dynamic.NewForConfigOrDie(resetConfig)
 	discoverClientSet := discovery.NewDiscoveryClientForConfigOrDie(resetConfig)
-
 	objectWatcher := objectwatcher.NewObjectWatcher(mgr.GetClient(), mgr.GetRESTMapper(), util.NewClusterDynamicClientSet)
 	overrideManager := overridemanager.New(mgr.GetClient())
-
+	resourceConfig := util.NewResourceConfig()
+	resourceConfig.Parse(opts.SkippedPropagatingAPIs)
```

Comment:
What if the input format is wrong? We should dump the error message and stop.

Reply:
yes, We should validate first

Reply:
Add a TODO